matroska_tags.xml

```diff
@@ -74,7 +74,7 @@ It **SHOULD** be a child of the following tags: "ARTIST", "LEAD_PERFORMER", or "
       <description lang="en">Email corresponding to the tag it's included in, using the "Addr-Spec" format defined in [@!RFC5322, section 3.4.1].</description>
     </tag>
     <tag name="ADDRESS" class="Nested Information" type="UTF-8">
-      <description lang="en">The physical address of the entity. The address **SHOULD** include a country code using the Country Code format defined in (#country-code-tags-formatting).
+      <description lang="en">The physical address of the entity. The address **MAY** include a country code using the Country Code format defined in (#country-code-tags-formatting).
```

Comment:
```suggestion
      <description lang="en">The physical address of the entity. The address **MAY** include a country code using the Country Code format defined in (#country-code-tags-formatting).
```
This one is a bit complicated. It's saying 

> **MAY** include a country code using the Country Code format defined in (#country-code-tags-formatting)

but I think you're saying something like 

> **MAY** include a country code.
> If a country code is included, the Country Code format defined in (#country-code-tags-formatting) is preferred.

Reply:
Yes it does make sense and avoids a potential confusion. 

I turned the requirement into a RECOMMENDED because existing tags may already use other formats. And there may be other, more standardized ways to define a physical address.